README.md

```diff
@@ -99,8 +99,12 @@ The AppId of the MSI will be displayed.
 The web app was successfully able to get a secret at runtime from Azure Key Vault using your developer account during development, and using MSI when deployed to Azure, without any code change between local development environment and Azure. 
 As a result, you did not have to explicitly handle a service principal credential to authenticate to Azure AD to get a token to call Key Vault. You do not have to worry about renewing the service principal credential either, since MSI takes care of that.  
 
-## Troubleshooting
+## How to use AzureCliCredential
```

Comment:
This is not enough info about AzureCliCredential

How to I install?
What does the code look like?  Does the code have to change?

Reply:
[Azure CLI 2.0]https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest has details about how to install Azure CLI credential on different systems. Codes are included in operating system's sub-directory.  
what about changing like below?

## How to use AzureCliCredential
1. Install Azure CLI 2.0, You can find how to install in [Azure CLI 2.0]https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest refer to different operating systems. If you have an earlier version, please upgrade. 
2. Once Azure CLI 2.0 installed. You can login using **az login** command in Windows Command Prompt or PowerShell.
3. Sign in with your account credentials in the browser.

Reply:
Resolved.

Reply:
You added comments on how to use the Azure CLI, which is OK, but please link to the new AzureCliCredential.  @tzhanl Can help.

Reply:
Resolved.